tests/acceptance/features/apiGraph/enforcePasswordPublicLink.feature

```diff
@@ -221,9 +221,9 @@ Feature: enforce password on public link
     Then the HTTP status code should be "200"
     And the OCS status code should be "200"
     And the OCS status message should be "OK"
-    And the public should not be able to download file "/textfile.txt" from inside the last public link shared folder using the new public WebDAV API without a password
-    And the public should not be able to download file "/textfile.txt" from inside the last public link shared folder using the new public WebDAV API with password "wrong pass"
-    But the public should be able to download file "/textfile.txt" from inside the last public link shared folder using the new public WebDAV API with password "<password>"
+    And the public should not be able to download file "/testfile.txt" from inside the last public link shared folder using the new public WebDAV API without a password
```

Comment:
it is because we are working on `testfile.txt` not `textfile.txt`. I got lucky to notice that because these are negative steps (so it would pass if we give any file)

Reply:
also, there is issue that accessing (downloading) public file with any name is possible (https://github.com/owncloud/ocis/issues/10343)